manga_auto_translator/pipeline.py

```diff
@@ -1,10 +1,12 @@
 from typing import Sequence
 from manga_auto_translator.data_structure import Scan
+from manga_auto_translator.ocr import OcrStrategyFactory
 
 
 class TranslationPipeline:
-    def __init__(self, scans: Sequence[Scan]) -> None:
+    def __init__(self, scans: Sequence[Scan], pipeline_args: dict) -> None:

```

Comment:
FYI : Dict from typing allow you to make a clean typing with python dictionnary. The format is the following :

```py
from typing import Dict

CUSTOM_TYPE = Dict[<type of key>, <type of value>]
```